src/modules/infobox/components/FeatureProfile.vue

```diff
@@ -192,6 +198,12 @@ export default {
             const id = this.feature.id.replace('drawing_feature_', '')
             this.deleteDrawingFeature(id)
         },
+        onCSVDownload() {
+            this.getProfile(profileCsv).then((data) => {
+                const dataBlob = new Blob([data], { type: 'text/csv', endings: 'native' })
+                this.triggerDownload(dataBlob, generateFilename('.csv'))
+            })
+        },
```

Comment:
it might be good to also `.catch(...)` here and log into the console any error (or transform that into an `async` function with a `try...catch` block)